@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -271,16 +306,21 @@ export class AltitudeBasedClipPlanesEvaluator implements ClipPlanesEvaluator {
 
         // Apply the constraints.
         nearPlane = Math.max(nearPlane - this.nearFarMargin / 2, this.nearMin);
-        farPlane = Math.min(farPlane, groundDistance * this.farMaxRatio);
+        farPlane = Math.min(farPlane, farMax);
         farPlane = Math.max(farPlane, nearPlane + this.nearFarMargin);
 
-        return { near: nearPlane, far: farPlane };
+        const viewRanges: ViewRanges = {
+            near: nearPlane,
+            far: farPlane,
+            maximum: Math.max(farMax, farPlane)
+        };
+        return viewRanges;
```

Comment:
and here

Reply:
We may do it in some other commit, as above it's not so crucial but will make a real rebasing mess (many dependent PR & commits).